docs/hacking.md

```diff
@@ -34,7 +34,7 @@ Then you can run the tests by running:
 
 You can alternatively run the tests in a more explicit fashion by passing the `KINTO_PSERVE_EXECUTABLE` environment variable, which can be found in the virtual environment's `bin` directory:
 
-    $ KINTO_PSERVE_EXECUTABLE="/path/to/venv/bin/pserve" npm run test
+    $ KINTO_PSERVE_EXECUTABLE="`pwd`/venv/bin/pserve" npm run test
```

Comment:
If I have kinto.js checked out in `/Users/Michiel/repos/kinto.js`, then venv command installs pserve in `/Users/Michiel/repos/kinto.js/venv/bin/pserve`, right? So then it's correct?